### ISSUE TITLE
When we converting css to React fontWeight should be in quotes for react native

### ISSUE THREAD

**Issue by ramazanarslan** [gottfired/css2react#5]
For React Native, font-weight:600;  should be converted in quotes `fontWeight:"600"`,  not     `fontWeight:600`. I have to change my code after converted. Can we add new properties for fontWeight.  
like :
```
{
   isFontWeightString:true 
}
```
thanks for this nice work

**Comment by gottfired** [comment]
Would fontWeight: "600px" also be ok? Because that would work in reactjs as well.

**Comment by ramazanarslan** [comment]
No, px is not suitable. İt should be  “600”. React native will give error if i use “600px”

**Comment by sabahmax-dev** [comment]
Thanks for the suggestion! I’ve opened PR #9 to add support for outputting `fontWeight` as a quoted string (e.g. `fontWeight: "600"`) when targeting React Native. This should address the issue in #5 and remove the need to manually adjust the converted code.
